### ISSUE TITLE
disable tests using obsolete UI framework on 6.4 till these are rewritten with AirGun

### ISSUE THREAD

**Issue by jhutar** [SatelliteQE/robottelo#6060]
At least these seems to still use old UI framework:

* tests.foreman.longrun.test_oscap
  *  OpenScapTestCase
     * test_positive_oscap_run_with_tailoring_file_and_capsule
     * test_positive_push_updated_content
     * test_positive_upload_to_satellite

Acked by Alex.

**Comment by san7ket** [comment]
Should adding stubbed marker to test case be ok , or is there any other way we can achieve this?

**Comment by jhutar** [comment]
@oshtaier please, could you answer above question? I'm pretty sure you know it... :-)

**Comment by oshtaier** [comment]
yes, we need to change jenkins configuration that specify what folders we run and say to ignore `longrun` folder

**Comment by san7ket** [comment]
#6104 Covers the stubbing of the tests, and let's have this issue as tracker to refactor those tests, so we don't miss them

**Comment by jhutar** [comment]
#6104 was merged. Thank you!

**Comment by jhutar** [comment]
Sorry, I have missed "let's have this issue as tracker to refactor those tests, so we don't miss them"
